lib/angular/portal.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = function(mainModule, userConfig) {
+  mainModule.config(function($stateProvider) {
+    if (userConfig && userConfig.mobile) {
+      return;
+    }
+
+    $stateProvider
+      .state('app.map', {
+        url: '/map',
+        resolve: {
+          workorders: function(workorderManager) {
```

Comment:
`workorderManager` depends on the manager implemented in the template apps.

I think module should have its own client to publish and subscribe to list workorders. 

E.g. Something like.

```
WorkorderMediatorService.prototype.listWorkorders = function listWorkorders() {
  var workorderListPromise = this.mediator.promise('done:wfm:workorders:list');

  this.mediator.publish('wfm:workorders:list');

  return workorderListPromise;
};
```